guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,388 @@
+4. The receiver identifies the payment as theirs by verifying that they control this address
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "."
+   width = 1024
+   height = 522
+   layout = "full-width"
+%}
+
+Silent payments introduces a new model based on new primitives/concepts that separates payment information based on its use. Below is a brief summary:
+
+| Silent payments' concept       | Purpose         | Sharing with |
+|:-------------|:------------------|:------|
+| Static address | Making payments | Senders
+| Scan key     | Payment scanning information  | Nodes & light clients
+| Spend key   | Spending bitcoin | Wallet owner
+
+The above model introduces a scan key that is used for payment scanning and a static address which is used for deriving on-chain addresses. This is in contrast with the BIP-32 model where [extended public keys](/guide/glossary/#extended-public-key-xpub-ypub-zpub) are used for both purposes. Below is a summary of the differences between the two models:
+
+| Current model with BIP-32       | Silent payments model          | Benefit  |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable & untraceable on-chain |
+| Extended private key (xprv) | Spend (private) key   |   |
+| Extended public key (xpub)           | Scan (public) key      | Allows scanning for payments but not derivation of addresses    |
+| **Address-based model**           | **Contact-based model** | **More intuitive and familiar**  |
+
+
+
+## Labels & contacts
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/wallet-privacy/network-settings-big.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+BIP-352 allows users to customize their static addresses on a case-by-case basis by adding labels. Users may add labels to customise their static addresses before sharing or posting them. When a payment is received, these labels can be detected and used to identify the static address used to make the payment without it being obvious on-chain.
```

Comment:
I would expect a couple of sentences about what labels are in general and how they are used today, before diving into the details of how labels work with silent payments and why they are useful.

Reply:
Done